soh/src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -96,7 +96,7 @@ u16 EnGo_GetTextID(GlobalContext* globalCtx, Actor* thisx) {
             if (gSaveContext.bgsFlag) {
                 return 0x305E;
             } else if (INV_CONTENT(ITEM_TRADE_ADULT) >= ITEM_CLAIM_CHECK) {
-                if (Environment_GetBgsDayCount() >= 3) {
+                if (Environment_GetBgsDayCount() >= CVar_GetS32("gForgeTime", 1)) {
```

Comment:
Since you registered `gForgeTime` with `3`, a value of `3` should be returned here.